h2o-core/src/main/java/water/MemKeyed.java

```diff
@@ -0,0 +1,31 @@
+package water;
+
+public class MemKeyed<T extends MemKeyed<T>> extends Keyed<T> {
```

Comment:
So this is something like `Chunk` if i understand well?

Reply:
this is the same pattern as chunk except for it also is Keyed which Chunk itself isn't